src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
+
+        System.out.println(ball.getRandomBall());
+
```

Comment:
![image](https://github.com/user-attachments/assets/488688e3-b6d2-49ea-b153-2afa8eeab8e2)
check requirements for this class

Reply:
When I created three balls, Ai said that I don't need to create three balls and in order to avoid mistakes I delated 2 balls, you can check this in my first commit